docs/nbbuild.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -euo pipefail
 
+# remove artifacts before building
+find docs -type f \( -name '*.html' -o -name '*-preview.png' \) -delete
```

Comment:
@shahinrostami heads up, so it doesn't surprise you!

Reply:
Thank you!